tests/flows/test_sync.py

```diff
@@ -1699,6 +1699,11 @@ def test_sync_agreement_prices_with_missing_prices(
     mock_adobe_client.get_subscriptions.assert_called_once_with("AUT-1234-5678", "a-client-id")
 
     assert mock_update_agreement_subscription.mock_calls == [
+        mocker.call(
+            mock_mpt_client,
+            terminated_mpt_subscription["id"],
+            template={"id": "TPL-1234", "name": "Renewing"},
```

Comment:
Why the template is renewing, while the changes you are making in `sync.py` are `expiring`?